src/test/resources/beforeTest.sql

```diff
@@ -1,45 +1,3 @@
-delete from parent_topic where uuid in ('94a10f9f-a42e-44c0-8ebe-1227cb347f1d',
```

Comment:
These lines were doing the same thing as those in afterTest.sql, but only partially. I've deleted them because it's confusing.